src/python/grpcio/grpc/_cython/loader.c

```diff
@@ -43,7 +42,7 @@ int pygrpc_load_core(char *path) { return 1; }
 // Cython doesn't have Py_AtExit bindings, so we call the C_API directly
 int pygrpc_initialize_core(void) {
   grpc_init();
```

Comment:
I think with this change we can completely remove this entire c loader hack.  If this is a quick fix for the import, feel free to drop a todo instead.